### PR TITLE
Nav bar about us 🖌️

### LINKED ISSUE #52: About Us -> Bottom Navigation Bar
### Is your feature request related to a problem? Please describe.
AboutUs Screen will consist of bottom navigation bar which will inflate 2 screens,

1. Aim Screen
2. Tab Screen
All the above screens should be ```Container( )``` for now .

### PR DESCRIPTION
**Pull request by TreavVasu**
### Description

Fix for #52

**Type of Change:** (Delete irrelevant options)
- Code
- User Interface

**Code/Quality Assurance Only** (Delete irrelevant options)

- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?

Describe the tests you ran to verify your changes. Provide instructions or GIFs so we can reproduce. List any relevant details for your test.

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] Any dependent changes have been merged
- [ ] I have added one/more required dependencies
- [x] I have formatted the code using dart format (dartfmt)

**Code/Quality Assurance**
- [x] My changes generate no new warnings 
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

**Mention the dependencies/packages/plugins that have been added (if any)**

### CODE REVIEW

**Comment by TreavVasu** [comment]
Would Resolve icons soon.

**Comment by TreavVasu** [comment]
Screenshot_1614809228
Screenshot_1614809232

**Review by werainkhatri — CHANGES_REQUESTED** [review]
1. navbar icons does not match with design.
2. add route to about us in menuscreen
3. merge main branch
4. remove pubspec.lock changes

**Comment by TreavVasu** [comment]
Screenshot_1615095242
Screenshot_1615095246

File: lib/screens/about_us/about_us.dart [hunk 1]
```diff
@@ -1,14 +1,100 @@
...
+  int _currentIndex = 0;
+  final tabs = [AimScreen(), TeamScreen()];
 
   @override
   Widget build(BuildContext context) {
-    return Container(
+    return Stack(
+      children: [
+        ScreenBackground(elementId: 0),
+        SafeArea(child: tabs[_currentIndex]),
```
Comment (werainkhatri) [inline lib/screens/about_us/about_us.dart]: Is there a need for SafeArea?

File: lib/screens/about_us/about_us.dart [hunk 2]
```diff
@@ -1,14 +1,100 @@
...
+class _AboutUsScreenState extends State<AboutUsScreen> {
+  int _currentIndex = 0;
+  final tabs = [AimScreen(), TeamScreen()];
 
   @override
   Widget build(BuildContext context) {
-    return Container(
+    return Stack(
+      children: [
+        ScreenBackground(elementId: 0),
```
Comment (werainkhatri) [inline lib/screens/about_us/about_us.dart]: remove ScreenBackground from here as there is ScreenBackground in Aim and Team

File: lib/screens/about_us/about_us.dart [hunk 3]
```diff
@@ -1,14 +1,100 @@
...
+              ),
+              BottomNavigationBarItem(
+                activeIcon: Container(
+                  height: 55,
+                  decoration: BoxDecoration(
+                      shape: BoxShape.circle,
+                      gradient: LinearGradient(colors: <Color>[
+                        Colors.orange[600],
+                        Colors.red[800],
+                        Colors.pink[300],
```
Comment (werainkhatri) [inline lib/screens/about_us/about_us.dart]: Copy the gradient from home.dart. This looks a bit off.

File: lib/screens/about_us/about_us.dart [hunk 4]
```diff
@@ -1,14 +1,100 @@
...
+                ),
+                icon: Image.asset(
+                  S.assetAIMIcon,
+                  height: 25,
+                ),
+                label: "Aim",
+              ),
+              BottomNavigationBarItem(
+                activeIcon: Container(
+                  height: 55,
```
Comment (werainkhatri) [inline lib/screens/about_us/about_us.dart]: set height to 50

**Review by werainkhatri — APPROVED** [review]
1. Revert pubspec.lock
2. give better asset file names.
3. see how bottom nav bar buttons are implemented in HomeScreen and do the same.

**Comment by werainkhatri** [comment]
Wrongly approved. Do the changes and I'll merge it.

**Comment by TreavVasu** [comment]
Screenshot_1615227485
Screenshot_1615227490

**Comment by werainkhatri** [comment]
No need to add images everytime you make changes. And do all the changes or talk to me if you have any doubt.

**Review by werainkhatri — APPROVED** [review]

**Pull request merged**